network/src/test/java/info/metadude/android/eventfahrplan/network/temporal/DateParserTest.kt

```diff
@@ -1,24 +1,43 @@
 package info.metadude.android.eventfahrplan.network.temporal
 
 import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatCode
 import org.junit.Test
 
 class DateParserTest {
 
     @Test
     fun `getDateTime date and time`() {
         assertThat(DateParser.getDateTime("2019-01-01T00:00:00Z")).isEqualTo(1546300800000)
+		assertThat(DateParser.getDateTime("1970-01-01T00:00:00Z")).isEqualTo(0)
+		assertThat(DateParser.getDateTime("1970-01-01T00:00:00+00:00")).isEqualTo(0)
+		assertThat(DateParser.getDateTime("1970-01-01T02:00:00+01:00")).isEqualTo(3600000)
+		
+		// test all formats of datetimes ever seen in schedule.xml files
+		assertThat(DateParser.getDateTime("2016-09-14T14:30:00+02:00")).isEqualTo(1473856200000)
+		assertThat(DateParser.getDateTime("2016-09-14T14:30:00+0200")) .isEqualTo(1473856200000)
```

Comment:
This raises an `org.threeten.bp.format.DateTimeParseException: Text '2016-09-14T14:30:00+0200' could not be parsed` to be thrown. Not sure if want to support this format. We should however have some mechanism to deal with unsupported formats. /cc @ideadapt

Reply:
I can have a deeper look in which schedules this format appears. Maybe they are handcrafted. 

The format can not be too far away from the standards, as it worked e.g. in vanilla Java 11 DateParser and I often see it in XML in completely other contexts. If something like a simple regex replace enables support for that format, it should be done IMO.

Reply:
We can certainly investigate how the ThreeTen library behaves different then Java 11 to find out if the string can be parsed in the app.
With regards to patching the input data I have strong feelings against doing this for the same reasons as stated below.

Reply:
See other conversations, it is very rare. I'd change this test case to be commented out, but still in the file for reference, does that make sense?

Reply:
Changed in e69b24c5cc278b32c5d86115e9a968af9d9c1db4. Please follow up in the conversation below. Closing here.